101-two-subnets/azuredeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/VNET.json",
```

Comment:
Doesn't look right. Needs to be "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",